automated install/basic-install.sh

```diff
@@ -62,7 +62,7 @@ else
   echo ":::"
   echo "::: Detecting the presence of the sudo utility for continuation of this install..."
 
-  if [ -x "$(command -v sudo)" ]; then
+  if command -v sudo; then
```

Comment:
Is that really equivalent?

<pre>
me@local ~ $ if command -v sudo; then
> echo "A";
> fi
<b>/usr/bin/sudo</b>
A
me@local ~ $ if [ -x "$(command -v sudo)" ]; then
> echo "A";
> fi
A
me@local ~ $
</pre>